README.md

```diff
@@ -235,6 +235,12 @@ Rust Gitter Channel: [![Gitter chat](https://badges.gitter.im/open-telemetry/ope
 
 Regular syncs take place every Thursday at 1:30 PM PT and are posted on the [calendar](https://github.com/open-telemetry/community#community-meetings). [Meeting notes](https://docs.google.com/document/d/1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY) are updated every sync.
 
+### Log SIG
+
+Log SIG meets biweekly on [TBD] via Zoom. Meeting notes are held in this [doc](https://docs.google.com/document/d/1cX5fWXyWqVVzYHSFUymYUfWxUK5hT97gc23w595LmdM/edit#).
```

Comment:
nit: biweekly is ambiguous, could be both twice a week or every other week but I'll assume it's the latter :)

Reply:
Changed to "every other week".